src/widget/wbeatspinbox.cpp

```diff
@@ -267,16 +269,27 @@ bool WBeatSpinBox::event(QEvent* pEvent) {
         updateTooltip();
     } else if (pEvent->type() == QEvent::FontChange) {
         const QFont& fonti = font();
-        qDebug() << "WBeatSpinBox::event QEvent::FontChange" << fonti.pixelSize() * m_scaleFactor;
         // Change the new font on the fly by casting away its constancy
         // using setFont() here, would results into a recursive loop
         // resetting the font to the original css values.
         // Only scale pixel size fonts, point size fonts are scaled by the OS
         if (fonti.pixelSize() > 0) {
             const_cast<QFont&>(fonti).setPixelSize(fonti.pixelSize() * m_scaleFactor);
```

Comment:
Is this needed anymore.

Thanks for working on this. This is an interesting work-around.